addons/binding/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/internal/convention/homeassistant/HandlerConfiguration.java

```diff
@@ -27,9 +29,31 @@
     /**
      * The MQTT prefix topic
      */
-    public String basetopic = "homeassistant";
+    private String basetopic = "homeassistant";
```

Comment:
please make this an immutable class. final public properties only. No getters.
Your empty constructor will then assign "homeassistant".

Reply:
I guess, my tendency towards getter/setters i showing my age ;-)